MapboxNavigation-SPM.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -24,17 +24,17 @@
         "repositoryURL": "https://github.com/mapbox/mapbox-common-ios.git",
         "state": {
           "branch": null,
-          "revision": "15221bf71677824975d2d9418fe76bd4259e9c3b",
-          "version": "21.2.1"
+          "revision": "59b81d4f6316a661c322e8c47bc3310f8a1bdd4a",
+          "version": "21.3.0"
```

Comment:
I do not have logical explanation why this issue happens on Xcode `v13.2.1` and doesn't on `v13.4.1`. Basically when I run `$ swift package update` with Xcode `v13.2.1` I get:
```shell
error:   Dependencies could not be resolved because root depends on 'mapbox-navigation-native-ios' 94.0.3..<95.0.0 and root depends on 'mapbox-maps-ios' 10.4.3..<11.0.0.
  'mapbox-navigation-native-ios' is incompatible with 'mapbox-maps-ios' because 'mapbox-navigation-native-ios' 94.0.3..<95.0.0 practically depends on 'mapbox-common-ios' 21.2.0..<22.0.0 (1).
  'mapbox-maps-ios' >= 10.4.3 practically depends on 'mapbox-common-ios' {21.2.1, 21.3.0, 22.0.0, 22.1.0} because 'mapbox-maps-ios' 10.6.0..<10.7.0-beta.1 depends on 'mapbox-common-ios' 22.0.0 and 'mapbox-maps-ios' >= 10.7.0 depends on 'mapbox-common-ios' 22.1.0.
  'mapbox-maps-ios' {10.4.3..<10.6.0, 10.7.0-beta.1..<10.7.0} practically depends on 'mapbox-common-ios' {21.2.1, 21.3.0} because 'mapbox-maps-ios' 10.5.0..<10.6.0-beta.1 depends on 'mapbox-common-ios' 21.3.0.
  'mapbox-maps-ios' {10.4.3..<10.5.0, 10.6.0-beta.1..<10.6.0, 10.7.0-beta.1..<10.7.0} practically depends on 'mapbox-common-ios' 21.2.1 because no versions of 'mapbox-maps-ios' match the requirement {10.5.0-beta.1..<10.5.0, 10.6.0-beta.1..<10.6.0, 10.7.0-beta.1..<10.7.0} and 'mapbox-maps-ios' 10.4.3..<10.5.0-beta.1 depends on 'mapbox-common-ios' 21.2.1.
  'mapbox-navigation-native-ios' 94.0.3..<94.0.5 practically depends on 'mapbox-common-ios' 21.2.0..<22.0.0 because 'mapbox-navigation-native-ios' 94.0.4..<94.0.5 depends on 'mapbox-common-ios' 21.2.0..<22.0.0 and 'mapbox-navigation-native-ios' 94.0.3..<94.0.4 depends on 'mapbox-common-ios' 21.2.1..<22.0.0.
     (1) As a result, 'mapbox-navigation-native-ios' 94.0.3..<95.0.0 practically depends on 'mapbox-common-ios' 21.2.0..<22.0.0 because 'mapbox-navigation-native-ios' 94.0.5..<95.0.0 depends on 'mapbox-common-ios' 21.2.1..<22.0.0.
```

When I run similar command using Xcode `v13.4.1` I get no errors and `Package.resolved` correctly updates dependencies.